nservicebus/dependency-injection/ninject.md

```diff
@@ -14,31 +14,28 @@ include: container-deprecation-notice
 
 NServiceBus can be configured to use [Ninject](http://www.ninject.org/) for dependency injection.
 
-
 ### Default usage
 
 snippet: Ninject
 
-
 ### Using an existing kernel
 
 snippet: Ninject_Existing
 
-
 ### Unit of work
 
-Its is possible to bind to use an _Unit of Work_ scope, which corresponds to the `DependencyLifecycle.InstancePerUnitOfWork` lifecycle, when registering components with `configuration.RegisterComponents(...)`.
+It is possible to bind to use an _Unit of Work_ scope, which corresponds to the `DependencyLifecycle.InstancePerUnitOfWork` lifecycle, when registering components with `configuration.RegisterComponents(...)`.
```

Comment:
@jpalac I'm reviewing this for editorial and this phrase stood out at me:

> It is possible to bind to use an _Unit of Work_ scope

My impression was you don't bind something "to use it". I don't have enough domain knowledge to know if it's right or wrong or suggest anything else, but figured I'd just follow up on it.

Reply:
Hmm - perhaps this will make more sense:

`It is possible to bind a service in a _Unit of Work_ scope,`

Reply:
Yes, thanks!